fireant/slicer/transformers/datatables.py

```diff
@@ -1,12 +1,14 @@
 # coding: utf-8
 import numpy as np
 import pandas as pd
+import locale as lc
 
 from datetime import time
 from fireant.slicer.operations import Totals
 from fireant import settings
 from fireant.slicer.transformers import Transformer
 
+lc.setlocale(lc.LC_ALL, 'en_US')
```

Comment:
Can you remove this hard-setting so it can be set externally?  This will affect everything that imports fireant